File: Java/smallestNum.java
import java.util.Scanner;

public class LabProgram {
   public static void main(String[] args) {
      Scanner scnr = new Scanner(System.in);
      int num1, num2, num3, smallestNum;
      
      num1 = scnr.nextInt();
      num2 = scnr.nextInt();
      num3 = scnr.nextInt();
      
      smallestNum = num1;
      
      if (num2 < smallestNum) {
         smallestNum = num2;
      }
      
      if (num3 < smallestNum) {
         smallestNum = num3;
      }
      
      System.out.println(smallestNum);
   }
}

File: C++/smallestNum.cpp
// Write a program whose inputs are three integers, and whose output is the smallest of the three values.
// Ex: If the input is: 7 15 3
//     the output is: 3

#include <iostream>
using namespace std;

int main() {
   
   int num1;
   int num2;
   int num3;

   cin >> num1;
   cin >> num2;
   cin >> num3;

   int smallestNum = num1;

   if (num2 < smallestNum) {
	   smallestNum = num2;
   }

   if (num3 < smallestNum) {
	   smallestNum = num3;
   }
   
   cout << smallestNum << endl;

   return 0;
}
